content/applications/finance/payment_acquirers/ogone.rst

```diff
@@ -5,84 +5,105 @@ Ogone
+You are now required to generate **SHA-IN** passphrases. **SHA-IN** and **SHA-OUT** passphrases are
+used to digitally sign the transaction requests and responses between Odoo and Ogone. By using these
+secret passphrases and the `sha-1` algorithm, both systems can ensure that the information they
+receive from the other was not altered or tampered with.
 
-.. _ogone/sha_key_out:
+Enter the same **SHA-IN** passphrase in both :guilabel:`Checks for e-Commerce & Alias Gateway` and
+:guilabel:`Checks for DirectLink and Batch (Automatic)`. You can leave the IP address field blank.
 
-SHA Key OUT
-~~~~~~~~~~~
+Your **SHA-IN** and **SHA-OUT** passphrases should be different, and between 16 and 32 characters
+long. Make sure to use the same **SHA-IN** and **SHA-OUT** passphrases throughout the entire Ogone
+configuration, as Odoo only allows a single **SHA-IN** and single **SHA-OUT** passphrase.
 
-In order to retrieve the SHA Key OUT, log into your ogone account, go to
-:menuselection:`Configuration --> Technical Information --> Transaction feedback --> All transaction
-submission modes`, and get or generate your **API Key** and **Client Key**. Be careful to copy your
-API key as you'll not be allowed to get it later without generating a new one.
+In order to retrieve the **SHA-OUT** key, log into your Ogone account, go to
+:menuselection:`Configuration --> Technical --> Information --> Transaction feedback --> All
+transaction submission modes`, and get or generate your API Key and Client Key. Be careful to copy
+your API key as you’ll not be
 
-.. important::
-   If you are trying Ogone as a test, with the Test Account, change the **State** to *Test Mode*. We
-   recommend doing this on a test Odoo database, rather than on your main database.
+When done, head to :menuselection:`Configuration --> Technical Information --> Transaction Feedback`
+and check the following options:
 
-Configuration on Ogone
-======================
+- :guilabel:`I would like to receive transaction feedback parameters on the redirection URLs`:
+  should be checked.
+- :guilabel:`Direct HTTP server-to-server request`: should to be set to `Online but switch to a
+  deferred request when the online request fails`.
+- Both **URL** fields should contain the same following URL, with `example` replaced by your
+  database: `https://<example>/payment/ogone/return`.
 
-Now that Odoo can communicate with Ogone, we need to make sure that Ogone can send information to
-your database.
+- :guilabel:`Dynamic eCommerce Parameters` should contain the following values: `ALIAS`, `AMOUNT`,
+  `CARDNO`, `CN`, `CURRENCY`, `IP`, `NCERROR` `ORDERID`, `PAYID`, `PM`, `STATUS`, `TRXDATE`. Other
+  parameters can be included (if you have another integration with Ogone that requires them), but
+  are not advised. The :guilabel:`URL` fields for :guilabel:`HTTP redirection in the browser` can be
+  left empty, as Odoo will specify these URLs for every transaction request.
```

Comment:
https://github.com/odoo/documentation/pull/2117#discussion_r904808420 was lost somewhere along the way.